Source/WebGPU/WGSL/Metal/MetalCodeGenerator.cpp

```diff
@@ -35,7 +35,7 @@ namespace WGSL {
 
 namespace Metal {
 
-static constexpr bool dumpMetalCode = true;
```

Comment:
why change this back? It seems useful so as long as WebGPU is disabled by default

Reply:
I had just accidentally enabled it before 😅, but if you found it useful I'm happy to keep it enabled

Reply:
Unless you think it will interfer with running the CTS

Reply:
I think it's not a problem. I kept it enabled for now, we can always turn it off later if it becomes an issue.